build.gradle

```diff
@@ -54,6 +54,9 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
     testImplementation group: 'junit', name: 'junit', version: '4.13.1'
 
+    // Lorem : An extremely useful Lorem Ipsum generator for Java!
+    implementation 'com.thedeanda:lorem:2.1'
```

Comment:
It seems this library is not actively maintained. 
@oboenikui Is this a de facto library for fake data generation? Do you know some alternative well-maintained libraries we can use?

Reply:
I'm unaware of de facto standard Lorem ipsum generator libraries, but I found two libraries that are more actively maintained.

- https://github.com/serpro69/kotlin-faker (most active)
- https://github.com/DiUS/java-faker (most stars)

I believe that kotlin-faker is more suitable because java-faker has several library vulnerabilities intact.

Although lorem is not actively maintained, it has the advantage of having no extra dependencies. However, the maintainer has apparently forgotten his Maven Central credentials, so if this continues, we may need to consider switching.

https://github.com/mdeanda/lorem/issues/16

Reply:
kotlin-faker looks a little heavy with its 17MB footprint (latest stable), while mdeanda's lorem is just 20KB

Reply:
any news here? I'd love to have the feature ;)

Reply:
In my opinion, there is no problem in continuing to use lorem because of the following points

* There is no shortage of functionality in lorem at the moment.
* The implementation itself is simple and there is no other library to depend on, so the risk of lorem not working due to future updates of IDEA is low.
* The current implementation is easy to replace with other libraries, so it can be replaced immediately if necessary.

As for kotlin-faker, upon closer examination, we found that it includes a dictionary of trademarks such as Nintendo in its resources, which could be problematic if it were to be published in the Marketplace. Therefore, we prefer java-faker if it is to be changed in the future.

If we need to change in the future, we can think of the following cases.

* When supporting multiple languages
* When more types of strings are to be supported

However, since java-faker is also currently stagnant in updates and depends on many libraries, we do not think we have the motivation to actively migrate to java-faker.